package.json

```diff
@@ -80,6 +80,7 @@
     "debug": "2.6.9",
     "edgedriver": "5.6.1",
     "error-stack-parser": "2.1.4",
+    "esbuild": "0.25.8",
```

Comment:
Direct dependency of "vite", which is already in dependencies

Reply:
is the version the same as in vite?

Reply:
vite depends on esbuild with "^" specifier, so it vite accepts any version